README.md

```diff
@@ -17,11 +17,42 @@ Analyze, Annotate, Publish. Turn documents into data.
+    -   This will build and start all of the docker images using docker-compose. It will bind to port 80 on localhost, so you must not have anything else running on port 80. The `local.yml` configuration file has the docker-compose details.
 5. Set `api.dev.documentcloud.org` and `minio.documentcloud.org` to point to localhost - `sudo echo "127.0.0.1 api.dev.documentcloud.org minio.documentcloud.org" >> /etc/hosts`
 6. Enter `api.dev.documentcloud.org/` into your browser - you should see the Django API root page.
-7. Install and run [Squarelet](https://github.com/muckrock/squarelet) and the [DocumentCloud frontend](https://github.com/muckrock/documentcloud-frontend) following the instructions in their repos to view the full-stack application.
+7. Install and run [Squarelet](https://github.com/muckrock/squarelet) and the [DocumentCloud frontend](https://github.com/muckrock/documentcloud-frontend) following the instructions in their repos to view the full-stack application. 
+    - `inv createsuperuser` in Squarelet (not DocumentCloud).
+    - Login using `inv shell` in Squarelet
+    - Set your user as `is_staff`:
+   ```
+   tempUser = User.objects.all()[0]
+   tempUser.is_staff = True
+   tempUser.save()
+   ```
+   - `make install` and `make dev` on the [DocumentCloud frontend](https://github.com/muckrock/documentcloud-frontend) to start the frontend (used at end).
+8. In Squarelet Django admin page, follow the instructions as for the ["Squarelet Integration" on MuckRock](https://github.com/muckrock/muckrock/#squarelet-integration), except:
+   - when creating the OpenID as set out in those instructions, also add `Redirect URIs`: `http://api.dev.documentcloud.org/accounts/complete/squarelet` and `http://minio.documentcloud.org/accounts/complete/squarelet`. 
+   - Set Post-logout Redirect UI: `http://dev.documentcloud.org`
+   - Scopes (one per line): 
+      ```
+      read_user
+      read_organization
+      read_auth_token
+      ```
+   - Add a Client profile and set the `Webhook URL`: `http://api.dev.documentcloud.org/squarelet/webhook/`. The `Source` can remain "MuckRock" in the dropdown.
+   - You will have to look up **three** security-related configuration lines from your Squarelet instance to DocumentCloud (as if it was MuckRock). 
+   - Write their values into the `./.envs/.local/.django` file of the DocumentCloud repository, which should already be initialized from above.
+   - `Client ID` goes into `SQUARELET_KEY`
+   - `Client Secret` goes into `SQUARELET_SECRET`
+   - Additionally, get the value for `JWT_VERIFYING_KEY` by opening the Squarelet Django shell using `inv shell` and copying the `settings.SIMPLE_JWT['VERIFYING_KEY']` (remove the leading `b'` and the trailing `'`, leave the `\n` portions as-is)
+9. Set `SIDEKICK_PROCESSING_URL=mock://sidekick.dev.documentcloud.org`
+10. Set `SQUARELET_WHITELIST_VERIFIED_JOURNALISTS=false` to allow you to login.
```

Comment:
These two can be added to `initialize_dotenvs.py`

Reply:
I believe this has been resolved on the latest commit